tests/integration_tests/scheduler/test_lsf_driver.py

```diff
@@ -114,15 +114,15 @@ async def test_lsf_info_file_in_runpath(runpath_supplied, tmp_path):
 async def test_job_name():
     driver = LsfDriver()
     iens: int = 0
-    await driver.submit(iens, "sleep 99", name="my_job_name")
+    await driver.submit(iens, "sleep 99", name="my_job")
```

Comment:
This is changed because "my_job_name" is too long for the standard bjobs output, it gets truncated..